src/kani-compiler/cbmc/src/goto_program/location.rs

```diff
@@ -46,15 +53,43 @@ impl Location {
         }
     }
 
+    pub fn function_name(&self) -> Option<InternedString> {
+        match self {
+            Location::Loc { function, .. } => *function,
+            _ => None,
+        }
+    }
+
+    pub fn get_column_number(&self) -> Option<u64> {
+        match self {
+            Location::Loc { col, .. } => *col,
+            _ => None,
+        }
+    }
+
+    pub fn get_comment(&self) -> Option<String> {
```

Comment:
CBMC tracks comments on source locations. Do we want to?

Reply:
We might only want it on asserts for now, so I'll change the assert properties instead.